includes/admin/class-edd-notices.php

```diff
@@ -132,6 +132,7 @@ public function add_notices() {
 
 		// User can manage the entire shop
 		if ( current_user_can( 'manage_shop_settings' ) ) {
+			$this->add_requirements_notices();
```

Comment:
I know this is part of the merge but should we open an issue to remove it in 3.0?

Reply:
I'm fine with just removing it in this branch. LOL I don't know why I even added it. Guess I wasn't really thinking about what I was doing.